cloud-sql-proxy.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: cloud-sql-proxy
+  version: 2.7.0
+  epoch: 0
+  description: The Cloud SQL Auth Proxy is a utility for ensuring secure connections to your Cloud SQL instances
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+  checks:
+    disabled:
+      - empty
```

Comment:
What's empty?

Reply:
You'll need to write the binary to ${{targets.destdir}} to get it in the package and you can remove this.

Reply:
this is the linter
the binary isn't placed in /usr/bin

Reply:
used go/build instead